Lib/asyncio/selector_events.py

```diff
@@ -28,6 +30,14 @@
 from . import trsock
 from .log import logger
 
+HAVE_SENDMSG = hasattr(socket.socket, 'sendmsg')
```

Comment:
Super nit: This variable name should probably start with `_`, like `_HAS_IPv6` in base_events.py.

Reply:
done